src/containers/PaymentSettingsPage/PaymentSettingsPage.duck.js

```diff
@@ -0,0 +1 @@
+// TODO
```

Comment:
At some point, this should contain a call `createStripeSetupIntent`

I was also thinking that a fetch for currentUser with a relationship: "stripeCustomer.defaultPaymentMethod" should be here. But maybe just a "wrapping" thunk function:

fetchCurrentUserWithPaymentMethod:
1) request: set state: fetchingPaymentMethod: 'inProgress'
=> user.duck.js :: fetchCurrentUser({relationship})
// fetchCurrentUser needs to be modified so that it can accept (optional) params object.
2) success: fetchingPaymentMethod: "succeeded"
3) error: fetchingPaymentMethod: "error"

Note: since currentUser is already fetched, we need to check if fetchCurrentUser is called with relationship stripeCustomer.defaultPaymentMethod.
 (Another option is to always fetch currentUser with the aforementioned relationship, but then it fetches unnecessary data ~90% of time.)